OPENSSL.md

```diff
@@ -0,0 +1,77 @@
+### Building OpenSSL for shell bundles
+
+To build OpenSSL for shell bundles, we require building:
```

Comment:
It's not obvious for others what a shell bundle is. A one line description would help a lot. I'd put it in the readme though.